web/themes/new_weather_theme/templates/layout/page--point.html.twig

```diff
@@ -80,7 +80,27 @@
               </div>
             </div>
           </div>
+          {% if  drupal_block("weathergov_weather_story_image", { wfo: point.grid.wfo }).content | length > 0  %}
           {{ drupal_block("weathergov_weather_story_image", { wfo: point.grid.wfo }) }}
+          {% else %}
```

Comment:
I think we should be able to store the first result in a variable, that way we don't call the block rendering function twice

```suggestion
          {% set weather_story =  drupal_block("weathergov_weather_story_image", { wfo: point.grid.wfo }) %}
          {% if  weather_story.content | length > 0  %}
          {{ weather_story }}
          {% else %}
```